apps/files/src/views/Settings.vue

```diff
@@ -137,39 +135,39 @@
 						<kbd>a</kbd>
 					</dt>
 					<dd class="shortcut-description">
-						{{ t('files', 'Open the actions menu for a file') }}
+						{{ t('files', 'File actions') }}
 					</dd>
 				</div>
 				<div>
 					<dt class="shortcut-key">
 						<kbd>F2</kbd>
 					</dt>
 					<dd class="shortcut-description">
-						{{ t('files', 'Rename a file') }}
+						{{ t('files', 'Rename') }}
 					</dd>
 				</div>
 				<div>
 					<dt class="shortcut-key">
 						<kbd>Del</kbd>
 					</dt>
 					<dd class="shortcut-description">
-						{{ t('files', 'Delete a file') }}
+						{{ t('files', 'Delete') }}
 					</dd>
 				</div>
 				<div>
 					<dt class="shortcut-key">
 						<kbd>s</kbd>
 					</dt>
 					<dd class="shortcut-description">
-						{{ t('files', 'Favorite or remove a file from favorites') }}
+						{{ t('files', 'Add or remove favorite') }}
```

Comment:
Is it possible in English to use the zeroarticle here? I mean instead of

```suggestion
						{{ t('files', 'Add or remove a favorite') }}
```

Reply:
yeah than I am fine with it - my German brain does not like the *zero article* ^^